tests/E2E/test/selectors/BO/advancedParameters/performence.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  Performence: {
```

Comment:
Performance*

Reply:
Thanks! it was a typo error :sweat_smile: